src/rred_reports/reports/report_config.toml

```diff
@@ -1,9 +1,16 @@
 [school]
-template = "input/templates/2022/2022-23_template.docx"
-masterfile = "input/processed/2022/masterfile_2022-23.xlsx"
-dispatch_list = "input/dispatch_lists/Dispatch list 2022-23.xlsx"
 n_tables = 7
 
+    [school.2022]
+    template = "input/templates/2022/2022-23_template.docx"
+    masterfile = "input/processed/2022/masterfile_2022-23.xlsx"
```

Comment:
should we just not make a year subfolder for these? Feels excessive as its only a single file per year